bcg6es-d1e110/cod-Zy765t_bcg6es-d1e110.xml

```diff
@@ -100,7 +100,7 @@
           <lb ed="#P" n="13"/>go, §. Nec expectandum est</ref> <!-- zum 2. Mal zitiert-->; sicut Dominus quaedam pericula Ec¬ 
           <lb ed="#P" n="14"/>clesiae suae in nouissimis diebus futura praedixit, <ref xml:id="bcg6es-d1e110-Rd1e275" corresp="#bcg6es-d1e110-Qd1e288">Matth. 24.</ref> 
           <lb ed="#P" n="15"/>vt Fideles Christi praemoniti vigilarent. Vnde ibidem super 
-          <lb ed="#P" n="16"/>illud, <quote xml:id="bcg6es-d1e110-Qd1e288" source="http://scta.info/resource/mt24_25" synch="1-3">Ecce praedixi vobis</quote>, <ref xml:id="bcg6es-d1e110-Rd1e291">dicit Glossa.</ref><!--Referenz zu Glosse, target wenn vorhanden--> <quote xml:id="bcg6es-d1e110-Qd1e294">Vt praescii vigiletis, praemo¬ 
+          <lb ed="#P" n="16"/>illud, <quote xml:id="bcg6es-d1e110-Qd1e288" source="http://scta.info/resource/mt24_25" synch="1-3">Ecce praedixi vobis</quote>, <ref xml:id="bcg6es-d1e110-Rd1e291" target="#bcg6es-d1e110-Qd1e294">dicit Glossa.</ref> <quote xml:id="bcg6es-d1e110-Qd1e294" source="http://scta.info/resource/go-mt24-d1e104" synch="5-7">Vt praescii vigiletis,</quote> <quote xml:id="bcg6es-d1e110-Qd1e295" source="http://scta.info/resource/go-mt24-d1e120" synch="7-8">praemo¬ <!-- ohne target? issue?-->
```

Comment:
This one is just about perfect. Just change `target="#bcg6es-d1e110-Qd1e294"` to `corresp="#bcg6es-d1e110-Qd1e294"`

If the `<ref>` corresponds to both of the gloss quotes below, you could add both values to the `@corresp`, separated by whitespace.

Something like `corresp="#bcg6es-d1e110-Qd1e294 bcg6es-d1e110-Qd1e295"`